.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
```

Comment:
this shouldn't be here. put it in .gitignore

Reply:
ok